node/bft/ledger-service/src/ledger.rs

```diff
@@ -35,18 +37,23 @@ use std::{
     },
 };
 
+/// The capacity of the LRU holiding the recently queried committees.
+const COMMITTEE_CACHE_SIZE: usize = 16;
```

Comment:
Is there a reason for using `16`?

This committee cache should just need the rounds near the ones being processed, but not sure what a good number would be.

Reply:
Not really; the idea was just that it would be small (for low overhead), but still large enough to cover all possible hits. It could probably be a bit smaller, but any resulting difference should be negligible.